### ISSUE TITLE
a question

### ISSUE THREAD

**Issue by xylovezxy** [muzairkhattak/multimodal-prompt-learning#53]
Hello, it's a pleasure to learn from your work. I have a question: I have my own dataset and I want to fine tune it on 1000 images. I want to test it on another 2000 images and use accuracy and recall as evaluation metrics. Which part of the code should I change?

**Comment by muzairkhattak** [comment]
Hi @xylovezxy,

Thank you for showing interest in MaPLe! 

Regarding your queries, kindly see the relevant closed issues related to fine-tuning the custom dataset. 

Kindly follow the instructions provided[ in this issue](https://github.com/muzairkhattak/multimodal-prompt-learning/issues/38), and [additionally this issue](https://github.com/muzairkhattak/multimodal-prompt-learning/issues/47) to fine-tune MaPLe on your dataset.

I hope that is helpful.
Thank you and kind regards!

**Comment by xylovezxy** [comment]
Thank you very much for your reply. I need to fine tune on 1000 images, but I still don't understand why the caltech 101 dataset needs to be set to train, val, and test. How to use all these 1000 images as the training set? Then test the precision and recall of the classification on another two thousand images

**Comment by muzairkhattak** [comment]
Hi @xylovezxy,

In that case, you can just keep the val information empty in the JSON file. It will then train on the training set and evaluate on the test set. 

Thank you and kind regards!

**Comment by xylovezxy** [comment]
Thank you very much for your reply and guidance. Wishing you all the best
